### PR TITLE
Settings: Changes in default settings

### PR DESCRIPTION
**Pull request by 64qam**
We've made some changes in the default settings as several application versions were obsolete (Maya 18, Nuke 11, PS 2020, etc). Also added tools and changed settings for Blender, Maya, and Blender. 

All should work as usual.

### CODE REVIEW

File: openpype/settings/defaults/project_anatomy/attributes.json [hunk 1]
```diff
@@ -10,16 +10,8 @@
...
-        "nukex/12-2",
-        "hiero/12-2",
-        "resolve/stable",
-        "houdini/18-5",
-        "blender/2-91",
-        "harmony/20",
-        "photoshop/2021",
-        "aftereffects/2021"
+        "maya/2024",
+        "nuke/14-0"
```
Comment (BigRoy) [inline openpype/settings/defaults/project_anatomy/attributes.json:14]: This suddenly seems like a **very limited default** with only maya and nuke visible for new projects for new users of this repository?

File: openpype/settings/defaults/project_anatomy/templates.json [hunk 2]
```diff
@@ -38,16 +38,6 @@
...
-        "simpleUnrealTextureHero": {
-            "folder": "{root[work]}/{project[name]}/{hierarchy}/{asset}/publish/{family}/hero",
-            "file": "{originalBasename}.{ext}",
-            "path": "{@folder}/{@file}"
-        },
-        "simpleUnrealTexture": {
-            "folder": "{root[work]}/{project[name]}/{hierarchy}/{asset}/publish/{family}/{@version}",
-            "file": "{originalBasename}_{@version}.{ext}",
-            "path": "{@folder}/{@file}"
-        },
```
Comment (BigRoy) [inline openpype/settings/defaults/project_anatomy/templates.json:50]: Were these unused?

File: openpype/settings/defaults/system_settings/tools.json [hunk 3]
```diff
@@ -1,90 +1,403 @@
...
-            "renderman": "Pixar Renderman"
+            "htoa": "Arnold for Houdini (example)",
+            "mtoa": "Arnold for Maya (example)",
+            "rendermanMaya": "Renderman for Maya (example)",
+            "yetiMaya": "Yeti for Maya (example)",
+            "redshiftMaya": "Redshift for Maya (example)",
+            "redshift3dsmax": "Redshift for 3dsmax (example)",
+            "mGear": "mGear for Maya (example)",
+            "vrayMaya": "Vray for Maya (example)",
+            "vraynuke": "Vray for Nuke (example)"
```
Comment (BigRoy) [inline openpype/settings/defaults/system_settings/tools.json:400]: Why do the maya equivalents have a capital `M` but the Nuke one doesn't? `vraynuke` versus `vrayMaya`?

File: openpype/settings/defaults/system_settings/tools.json [hunk 4]
```diff
@@ -1,90 +1,403 @@
 {
     "tool_groups": {
+        "htoa": {
+            "environment": {
+                "HOUDINI_PATH": [
+                    "{STUDIO_SW}/APP/HTOA/{HTOA_VERSION}/HOUDINI{HOUDINI_VERSION}/WINDOWS/htoa-6.1.3.3_rdb15014_houdini-{HTOA_VERSION}",
```
Comment (BigRoy) [inline openpype/settings/defaults/system_settings/tools.json:6]: If we're mapping all of these to `{STUDIO_SW}` as an example should we also have an example default `STUDIO_SW` defined in the global env?

File: openpype/settings/defaults/project_settings/maya.json [hunk 5]
```diff
@@ -1,5 +1,5 @@
 {
-    "open_workfile_post_initialization": false,
+    "open_workfile_post_initialization": true,
```
Comment (BigRoy) [inline openpype/settings/defaults/project_settings/maya.json:2]: Why are we making this the default?

File: openpype/settings/defaults/project_anatomy/templates.json (hunk 2 above)
Comment (BigRoy) [reply openpype/settings/defaults/project_anatomy/templates.json:50]: There seems to be quite some more usage of "simpleUnrealTexture" in the code base?

**Review by simonebarbieri — APPROVED** [review]
Blender settings changes make sense. The new default version make sense too. Maybe 3.3 could be added as well, since that one is also LTS, but 3.6 should be more than enough!

File: openpype/settings/defaults/project_anatomy/templates.json (hunk 2 above)
Comment (antirotor) [reply openpype/settings/defaults/project_anatomy/templates.json:50]: True, this PR #6012 should eradicate it completely

File: openpype/settings/defaults/project_settings/maya.json (hunk 5 above)
Comment (antirotor) [reply openpype/settings/defaults/project_settings/maya.json:2]: This is solving issues with maya plugin init when opening some scenes and it doesn't introduce any additional side effects, so I guess it is better to have it on by default.

File: openpype/settings/defaults/system_settings/tools.json (hunk 4 above)
Comment (antirotor) [reply openpype/settings/defaults/system_settings/tools.json:6]: That we should definitely do

**Review by antirotor — DISMISSED** [review]

File: openpype/settings/defaults/project_anatomy/attributes.json (hunk 1 above)
Comment (antirotor) [reply openpype/settings/defaults/project_anatomy/attributes.json:14]: or better, add latest version of everything we support in Applications addon defaults and turn on "show only available"

File: openpype/settings/defaults/project_settings/maya.json (hunk 5 above)
Comment (BigRoy) [reply openpype/settings/defaults/project_settings/maya.json:2]: It was only intended for XGEN right?

File: openpype/settings/defaults/project_anatomy/attributes.json (hunk 1 above)
Comment (mkolar) [reply openpype/settings/defaults/project_anatomy/attributes.json:14]: Agreed 100% with Ondrej here

File: openpype/settings/defaults/system_settings/tools.json (hunk 4 above)
Comment (mkolar) [reply openpype/settings/defaults/system_settings/tools.json:6]: agreed

**Comment by mkolar** [comment]
> Blender settings changes make sense. The new default version make sense too. Maybe 3.3 could be added as well, since that one is also LTS, but 3.6 should be more than enough!

I'd keep to the latest only. The slimmer the settings the better

**Comment by mkolar** [comment]
I've converted the changes to AYON. OpenPype defaults are staying, there's no point touching those at this point.

File: openpype/settings/defaults/system_settings/tools.json (hunk 3 above)
Comment (mkolar) [reply openpype/settings/defaults/system_settings/tools.json]: That's not actually being touched here at the end. So resolving conversations....to answer though. I'd say just an accident.

**Pull request merged**